<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BiosTool</class>
 <widget class="QDialog" name="BiosTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>699</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>BiosTool</string>
  </property>
  <widget class="QTabWidget" name="Gset_Analysis">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>19</y>
     <width>821</width>
     <height>641</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>BIOS Logfile Analysis </string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>471</width>
       <height>261</height>
      </rect>
     </property>
     <property name="title">
      <string>Input</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>POST LogFile:</string>
        </property>
        <property name="buddy">
         <cstring>logfile_name</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="logfile_name">
        <property name="placeholderText">
         <string>LivingStone2.log</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>37</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="5">
       <widget class="QPushButton" name="file_browser">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icon/loupe.png</normaloff>icon/loupe.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="root_path">
        <property name="placeholderText">
         <string>project root path</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="folder_browser">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icon/search_query_128px_1063400_easyicon.net.ico</normaloff>icon/search_query_128px_1063400_easyicon.net.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="5">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Project Location:</string>
        </property>
        <property name="buddy">
         <cstring>root_path</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="Execute">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>289</y>
       <width>481</width>
       <height>274</height>
      </rect>
     </property>
     <property name="title">
      <string>Execute</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>46</y>
        <width>211</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>false</italic>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Refresh Guid Template</string>
      </property>
      <property name="buddy">
       <cstring>e_refresh_guid_template</cstring>
      </property>
     </widget>
     <widget class="QPushButton" name="e_refresh_guid_template">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>26</y>
        <width>62</width>
        <height>58</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icon/cloud-computing.png</normaloff>icon/cloud-computing.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>123</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>false</italic>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Replace LogFile Guid</string>
      </property>
      <property name="buddy">
       <cstring>e_replace_logfile_guid</cstring>
      </property>
     </widget>
     <widget class="QPushButton" name="e_replace_logfile_guid">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>103</y>
        <width>62</width>
        <height>58</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icon/exchange.png</normaloff>icon/exchange.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>311</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>false</italic>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Produce Driver Loading Sequence</string>
      </property>
      <property name="buddy">
       <cstring>e_produce_driver_loading_sequence</cstring>
      </property>
     </widget>
     <widget class="QPushButton" name="e_produce_driver_loading_sequence">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>180</y>
        <width>62</width>
        <height>58</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icon/timeline.png</normaloff>icon/timeline.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>300</y>
       <width>256</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>9</y>
       <width>256</width>
       <height>274</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>20</y>
       <width>251</width>
       <height>261</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icon/cat-icon.png</normaloff>icon/cat-icon.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>400</width>
       <height>400</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Gset">
    <attribute name="title">
     <string>Gset Analysis</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>481</width>
       <height>261</height>
      </rect>
     </property>
     <property name="title">
      <string>Input</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="2">
       <widget class="QLineEdit" name="root_path_2">
        <property name="placeholderText">
         <string>c:\LivingStone2</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <spacer name="horizontalSpacer_13">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="root_path_3">
        <property name="placeholderText">
         <string>c:\LivingStone2\OEMBOARD\LivingStone2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_11">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_10">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Platform Folder Location:</string>
        </property>
        <property name="buddy">
         <cstring>root_path</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="folder_browser_3">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icon/search_query_128px_1063400_easyicon.net.ico</normaloff>icon/search_query_128px_1063400_easyicon.net.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="horizontalSpacer_12">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="folder_browser_2">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icon/search_query_128px_1063400_easyicon.net.ico</normaloff>icon/search_query_128px_1063400_easyicon.net.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Output Location:</string>
        </property>
        <property name="buddy">
         <cstring>root_path</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Source Code Location:</string>
        </property>
        <property name="buddy">
         <cstring>root_path</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="horizontalSpacer_14">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="3">
       <spacer name="horizontalSpacer_15">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="root_path_4">
        <property name="placeholderText">
         <string>c:\temp</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QPushButton" name="folder_browser_4">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icon/search_query_128px_1063400_easyicon.net.ico</normaloff>icon/search_query_128px_1063400_easyicon.net.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_2">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>310</y>
       <width>251</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="Execute_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>433</y>
       <width>481</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>Execute</string>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>46</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>false</italic>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Produce Gset items Excel file </string>
      </property>
      <property name="buddy">
       <cstring>e_refresh_guid_template</cstring>
      </property>
     </widget>
     <widget class="QPushButton" name="e_produce_gset_items_excel">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>30</y>
        <width>62</width>
        <height>58</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icon/timeline.png</normaloff>icon/timeline.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>40</y>
       <width>251</width>
       <height>251</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icon/cat-icon.png</normaloff>icon/cat-icon.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>400</width>
       <height>400</height>
      </size>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>481</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>Setup Variable</string>
     </property>
     <widget class="QLineEdit" name="root_path_5">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>90</y>
        <width>161</width>
        <height>30</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>c:\temp</string>
      </property>
     </widget>
     <widget class="QPushButton" name="folder_browser_5">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>80</y>
        <width>62</width>
        <height>52</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icon/search_query_128px_1063400_easyicon.net.ico</normaloff>icon/search_query_128px_1063400_easyicon.net.ico</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Input Location:</string>
      </property>
      <property name="buddy">
       <cstring>root_path</cstring>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>271</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>used runtime setup variable</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>root_path</tabstop>
  <tabstop>folder_browser</tabstop>
  <tabstop>logfile_name</tabstop>
  <tabstop>file_browser</tabstop>
  <tabstop>e_refresh_guid_template</tabstop>
  <tabstop>e_replace_logfile_guid</tabstop>
  <tabstop>e_produce_driver_loading_sequence</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>plainTextEdit</tabstop>
  <tabstop>root_path_2</tabstop>
  <tabstop>folder_browser_2</tabstop>
  <tabstop>root_path_3</tabstop>
  <tabstop>folder_browser_3</tabstop>
  <tabstop>root_path_4</tabstop>
  <tabstop>folder_browser_4</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>root_path_5</tabstop>
  <tabstop>folder_browser_5</tabstop>
  <tabstop>e_produce_gset_items_excel</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>plainTextEdit_2</tabstop>
  <tabstop>Gset_Analysis</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
